src/core_ocean/shared/mpas_ocn_tracer_short_wave_absorption_jerlov.F

```diff
@@ -186,6 +190,23 @@ subroutine ocn_tracer_short_wave_absorption_jerlov_tend(meshPool, forcingPool, i
       !$omp end parallel
 #endif
 
+      if(config_enable_shortwave_energy_fixer) then
+#ifndef CPRPGI
+         !$omp parallel
+         !$omp do schedule(runtime) private(k, layerFraction)
+#endif
+         do iCell = 1, nCells
+            layerFraction = bottomlayerShortwaveTemperatureFlux(iCell) / (maxLevelCell(iCell) + 1.0E-15_RKIND)
+            do k=1,maxLevelCell(iCell)
```

Comment:
@mark-petersen does this seem safe against the future case of `maxLevelcell=0`?

Reply:
Yes.